### PR TITLE
Add Drop Table and API endpoint

### PR DESCRIPTION
**Pull request by savage13**
Add Drop Table data and create API endpoints 
  - `/drop/:actor_name/:table_name`
  - `/drop/:actor_name`

### CODE REVIEW

File: build.ts [hunk 1]
```diff
@@ -299,6 +309,18 @@ function processBanc(filePath: string, mapType: string, mapName: string) {
           }
         }
       }
+      // If DropTable and DropActor do not exist and an blank drop exists
+      //   set the DropTable name to as 'Default'
+      if (!dyn.Drop__DropTable && !dyn.Drop__DropActor) {
+        if (actor.Gyaml in DROP_TABLES) {
+          for (const table of DROP_TABLES[actor.Gyaml]) {
+            if (table.DropTableName == "") {
+              drops.push(2);
```
Comment (leoetlino) [inline build.ts]: what is the 2? and can we get rid of it?

File: drop_table_read.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
```
Comment (leoetlino) [inline drop_table_read.py:1]: could you reformat with e.g. `black` please?

File: build.ts (hunk 1 above)
Comment (savage13) [reply build.ts]: I added `const`s for `DropTable = 2` and `DropActor = 1`.  We can change them to strings if desired.  Those values are also on the front-end.

File: drop_table_read.py (hunk 2 above)
Comment (savage13) [reply drop_table_read.py:1]: Done

File: build.ts (hunk 1 above)
Comment (leoetlino) [reply build.ts]: I think this makes parsing rather awkward - can we turn drops into an array of `{type: DROP_TABLE / DROP_ACTOR, value: xxx}` objects?

Comment (leoetlino) [reply build.ts]: also we probably want to give those constants a prefix, e.g. DROP_TYPE_DROP_TABLE and DROP_TYPE_ACTOR

Comment (savage13) [reply build.ts]: Shape of drops has changed to 
`{ type: string, value: string[] }`
- type is either "Actor" or "Table"
- value is [ table_name ] when type = Table
- value is [ actor, actor ... ] when type = Actor

**Pull request merged**